src/_cffi_src/openssl/x509.py

```diff
@@ -258,6 +259,12 @@
 int i2d_re_X509_CRL_tbs(X509_CRL *, unsigned char **);
 void X509_REQ_get0_signature(const X509_REQ *, const ASN1_BIT_STRING **,
                              const X509_ALGOR **);
+X509_ATTRIBUTE *X509_REQ_get_attr(const X509_REQ *, int);
```

Comment:
You have these grouped with 1.1.0 functions. Since these aren't, please move or seperate.